src/OSPSuite.InstallationValidator.Core/Presentation/MainPresenter.cs

```diff
@@ -35,14 +39,19 @@ public void SelectOutputFolder()
          _outputFolderDTO.FolderPath = outputFolder;
       }
 
+      public override void ViewChanged()
+      {
+         _view.OkEnabled = _view.HasError;
+      }
+
       public void Abort()
       {
-         throw new System.NotImplementedException();
+         _batchStarterTask.StopValidation(_process);
```

Comment:
See above. Use cancelationToken and stop the task etc.. see ParmaeterIdentificationRun in Core I think